react/features/chat/components/web/ChatInput.tsx

```diff
@@ -170,10 +173,20 @@ class ChatInput extends Component<IProps, IState> {
      * @returns {void}
      */
     _onSubmitMessage() {
+        const {
+            _isSendGroupChatDisabled,
+            _privateMessageRecipientId,
+            onSend
+        } = this.props;
+
+        if (_isSendGroupChatDisabled && !_privateMessageRecipientId) {
```

Comment:
Why the check here if the button will be disabled in this case?

Reply:
because it can be triggered through keyboard shortcut as well

Reply:
Oh.